webapi/src/main/scala/org/knora/webapi/responders/v1/ResourcesResponderV1.scala

```diff
@@ -1176,66 +1176,6 @@ class ResourcesResponderV1 extends Responder {
 
 
     /**
-      * Implements a pre-update check to ensure that an [[UpdateValueV1]] has the correct type for the `knora-base:objectClassConstraint` of
```

Comment:
I added user profile to some of the case classes while refactoring ontology responder, see `ResourcesResponderV1.scala` here https://github.com/dhlab-basel/Knora/pull/478/files#diff-13eb43a5343d8a8f59b7bd0a75fa4350